### ISSUE TITLE
Self Contained Vaults throw initialization errors with latest Dendron version, but still function (v.100)

### ISSUE THREAD

**Issue by ScriptAutomate** [dendronhq/dendron#3122]
## Please select the area your bug applies to. (Multiple selections are Possible. You can leave blank if you're not sure.)

- [x] Workspace.  VSCode workspace, vaults, Intellisense/autocomplete, Dendron settings
- [ ] Lookup.  Dendron's Lookup Command
- [ ] Views.  Dendron Preview, Tree View, Side Panels in the UI
- [ ] Schema.  Dendron Schemas
- [ ] Pod.  Data import from / export to Dendron
- [ ] Publish.  External Site Publish
- [ ] Markdown.  Markdown features, such as syntax support and features

## Describe the bug

I've run into an issue with the latest Dendron release (v100) and Self Contained Vaults. Dendron throws initialization errors when opening Self Contained Vaults in the root of their workspaces, and they only seem to go away if I open the dendron.code-workspaces file. When I open that file, though, I am no longer able to see the root of my project vault dir (where things like dendron.yml exist), as the file explorer only shows the contents of the notes/* directories of my vault and dependency vaults.

Funny thing though: I am getting both an initialization issue error output, and a `Dendron is active` output, when opening the root dir of my Self Contained Vault. As far as I can tell, I think Dendron is still functioning properly. I don't get any errors if I choose to open the workspace file directly instead of directory, but that results in the File Explorer layout I mentioned above.

## To Reproduce

1. Initialize a new Self Contained Vault
2. `cd` into the root of the Self Contained Vault
3. Open with `code .` (or `codium .`)
4. See errors, but also see success message in initialization
  - `issue initializing Dendron`
  - `{}`
  - `Dendron is active`

## Expected behavior

Prior to upgrading to v.100, no errors were being thrown. The vault seems to still function as expected.

## Screenshots

![image](https://user-images.githubusercontent.com/5951023/174894373-fa38876f-c55c-48a6-b8a4-e2024273f136.png)

## Desktop (please complete the following information)

- OS: Pop!_OS 22.04 (Linux)
- Versions (all that apply to bug):
  - Dendron Extension: `v0.100.0`
  - VS Code version: `1.68.1` (VS Code)

## Dendron Log file

Please attach the output of `> Dendron:Dev: Open Logs` here

> Here is a snippet of the error output in the logs

```
{"level":30,"time":1655829031911,"pid":23441,"hostname":"pop-os","name":"dendron","ctx":"activateWorkspace","stage":"prod","msg":"enter"}
{"level":30,"time":1655829031919,"pid":23441,"hostname":"pop-os","name":"dendron","ctx":"togglePluginActiveContext","state":"togglePluginActiveContext: true"}
{"level":50,"time":1655829031922,"pid":23441,"hostname":"pop-os","name":"dendron","stack":"Error: no workspace file\n\tat Function.workspaceFile (/home/myuser/.vscode/extensions/dendron.dendron-0.100.0/dist/extension.js:8:3135643)\n\tat Ca (/home/myuser/.vscode/extensions/dendron.dendron-0.100.0/dist/extension.js:8:3393067)\n\tat processTicksAndRejections (node:internal/process/task_queues:96:5)","type":"Error","msg":"no workspace file"}
{"level":50,"time":1655829031924,"pid":23441,"hostname":"pop-os","name":"dendron","ctx":"showInitProgress","msg":"issue initializing Dendron"}
{"level":30,"time":1655829032103,"pid":23441,"hostname":"pop-os","name":"dendron","ctx":"updateDecorations","payload":{"error":null,"decorationsLength":4,"diagnosticsLength":0}}
```

**Comment by SeriousBug** [comment]
Thanks for the report, I'll look into this!

**Comment by ScriptAutomate** [comment]
This issue seems to be resolved while running v0.100.1 :) Going to close this issue.
